systemtest/src/test/java/io/strimzi/systemtest/KafkaClusterTest.java

```diff
@@ -213,4 +215,30 @@ private void waitForZkMntr(String pod, Pattern pattern) {
         });
     }
 
+    @Test
+    @KafkaCluster(name = "my-cluster",
+            cmConfiguration = {@KafkaCluster.CmConfiguration(key = "zookeeper-healthcheck-delay", value = "30"),
+                    @KafkaCluster.CmConfiguration(key = "zookeeper-healthcheck-timeout", value = "10"),
+                    @KafkaCluster.CmConfiguration(key = "kafka-healthcheck-delay", value = "30"),
+                    @KafkaCluster.CmConfiguration(key = "kafka-healthcheck-timeout", value = "10"),
+                    @KafkaCluster.CmConfiguration(key = "KAFKA_DEFAULT_REPLICATION_FACTOR", value = "2"),
+                    @KafkaCluster.CmConfiguration(key = "KAFKA_OFFSETS_TOPIC_REPLICATION_FACTOR", value = "5"),
+                    @KafkaCluster.CmConfiguration(key = "KAFKA_TRANSACTION_STATE_LOG_REPLICATION_FACTOR", value = "5")
+            })
+    public void testClusterWithCustomParameters() {
```

Comment:
This isn't making the ideal assertions. The `StrimziRunner` will process the `@KafkaCluster` and create the CM, so asserting on the CM values is really only testing the StrimziRunner. What you should really be asserting on is that the pods that get created by the CC have the matching replication factors, etc. That's a little fiddly to do right now because we don't have external access to the Kafka brokers -- you've have to use `client.exec()` to run `kafka-configs.sh` inside the pod and then parse the output. But doing that you're actually testing that the CC deploys a SS with the given configuration, which is a much stronger test.

Reply:
I found an easy way to get this values, it can be the execution of command: "oc get <object_type> [<object_name_or_id>] -o json". In this JSON we have all necessary information for the pod in readable form:
            "env": {
                        "name": "KAFKA_DEFAULT_REPLICATION_FACTOR",
                        "value": "2"
...
                "readinessProbe": {
                    "exec": {
                        "command": [
                            "/opt/kafka/kafka_healthcheck.sh"
                        ]
                    },
                    "initialDelaySeconds": 30,
                    "periodSeconds": 10,
                    "timeoutSeconds": 10
                },
...
Would it be enough? WDYT?
JSON with response from OC - [POD_JSON.txt](https://github.com/strimzi/strimzi/files/1798296/JSON.txt)

Reply:
After discussion with @tombentley we decided that the checking of env variables is not a good idea because in future Strimzi will not use these env variables. As a conclusion, I created a new issue https://github.com/strimzi/strimzi/issues/310 to create assertions for Broker configuration and added TODO in the code where QE need to check configuration on Kafka brokers.